### ISSUE TITLE
[help] Logo

### ISSUE THREAD

**Issue by swyxio** [sw-yx/ssg#9]
ssg needs a logo before launch

theres a small chance that ssg becomes framework agnostic, so we may not want to tie to svelte too much, altho ofc svelte will forever be first class

**Comment by m4rrc0** [comment]
Do we want to keep the name as 'Sapper Site Generator' knowing that svelte might becomes optional at some point?

Maybe you thought was just to avoid mentioning the full name in the logo anyway so we just have to find another word beginning with 's' for the first word if we decouple from svelte and sapper? ^^

Do you have anything at all in mind for that logo? colors, other logos you like, ...?

What would you consider being the core values and principles that you want to carry with ssg?

**Comment by swyxio** [comment]
hey! haha thanks for engaging. you're right, may not want to have a hard dependency on svelte and sapper.

no idea for logo, probably just letters in a nice font? unless we have a mascot we like

ssg values - speed, simplicity - thats all i can think of right now.. (please feel free to add more opinions!)

**Comment by Nuhvi** [comment]
#20 

Quick concept
## concept

Gatsby >> Circular logo >> Infinite vertices
Hugo >> Hexagonal logo >> 6 vertices
11ty simpler ssg >> square logo >> 4 vertices

this >> could be a triangle >> 3 smallest number of vertices for an area! 

Triangles waste a lot of square logo real estate:
![ssg-logo00](https://user-images.githubusercontent.com/40009100/81043900-e29a1a00-8ebb-11ea-929f-bff556f308a5.png)


Second best thing: **Reuleaux triangle**!

1- fills more of the logo: 
![ssg-logo01](https://user-images.githubusercontent.com/40009100/81043953-f776ad80-8ebb-11ea-88c4-d221dc704974.png)

2- made with by intersection of 3 circles (simplicity, speed, whatever else)
![ssg-logo02](https://user-images.githubusercontent.com/40009100/81043784-a9fa4080-8ebb-11ea-8a8e-9e1ae614bb43.png)

3- Deceivingly simple, as it can roll (dynamic/powerful)!
https://www.youtube.com/watch?v=cUCSSJwO3GU

## Construction

![ssg-logo03](https://user-images.githubusercontent.com/40009100/81044410-f2662e00-8ebc-11ea-8a08-51a059756d31.png)

## Result
![ssg-logo](https://user-images.githubusercontent.com/40009100/81043214-87b3f300-8eba-11ea-8389-9235bc4fde7d.png)

Edit: small change for my peace of mind

![ssg-logo04](https://user-images.githubusercontent.com/40009100/81052763-b5099c80-8ecc-11ea-9f00-db83397c693a.png)

**Comment by swyxio** [comment]
whoooaooaaooaoaoaoaoaoa thank you! this looks nice!

so currently this project is dormant as i am writing my book right now, but i still use ssg actively for my personal site and am close to the point of needing to rewrite it.. at which point i will adopt a logo and push towards a v1. thank you so much! love the thinking behind it, wish there was some way to fit the "ssg" letters in there but it would probably be too complex love this shield-looking Reuleaux format!

**Comment by Nuhvi** [comment]
Glad you liked it!

If you squint you can see the SSG in the third dimension :D /s

![ssg-logo05](https://user-images.githubusercontent.com/40009100/81076083-e21d7580-8ef3-11ea-82c7-fadb72fc7f8a.png)


I am aware of the status of the project and it's why I didn't venture to use it instead of Gatsby in my second iteration of my personal site!

I am very curious to learn about your take on SSGs and how would you do things differently than Gatsby and 11ty, and if I can contribute to it!

Maybe once you ship your book we might have a chat/talk/record podcast, where you explain your plans!

Good luck!

**Comment by swyxio** [comment]
yup! the basic idea is given in this blogpost https://www.swyx.io/writing/bad-ways-to-get-data (why we need a different way to fetch data than gatsby/every other ssg) and this talk https://youtu.be/JEH0V-zNGUk (why sapper isnt good enough)

**Comment by Nuhvi** [comment]
I went through them, but I am more curious about how SSG differs from Gatsby/11ty/Nextjs than from Sapper. I have a lot of reading to do about how different projects approach the same problem. I will try to have better questions by the time you finish the book!
